pyproject.toml

```diff
@@ -25,31 +25,31 @@ keywords = [
     'brain',
 ]
 classifiers = [
+    'Intended Audience :: Developers',
+    'Intended Audience :: Science/Research',
+    'License :: OSI Approved :: BSD License',
+    'Natural Language :: English',
+    'Operating System :: MacOS',
     'Operating System :: Microsoft :: Windows',
     'Operating System :: Unix',
-    'Operating System :: MacOS',
     'Programming Language :: Python :: 3 :: Only',
     'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
     'Programming Language :: Python :: 3.11',
-    'Natural Language :: English',
-    'License :: OSI Approved :: BSD License',
-    'Intended Audience :: Science/Research',
-    'Intended Audience :: Developers',
     'Topic :: Scientific/Engineering',
 ]
 dependencies = [
-    'numpy>=1.21',
-    'scipy',
-    'mne>=1.1.0',
+    'decorator',
+    'importlib-resources; python_version < "3.9"',
+    'jinja2',
     'joblib',
     'matplotlib',
-    'scikit-learn',
+    'mne>=1.2',
```

Comment:
Bumping MNE minimum requirement to 1.2, since this will be released with MNE 1.6, and keeping compatibility with the 4 last versions of our core dependencies is good practice.